src/bot/commands/settings.service.ts

```diff
@@ -69,12 +69,16 @@ export class SettingsService {
           // const extension = attachment.name?.split('.').at(-1) ?? '';
           const norm_file = await this.audio.normalizeLoudness(file);
           const uri = await this.storage.storeFile('opus', hash, norm_file);
+          const intro_duration = await this.audio.getTrackDuration(file);
+          const intro_timeout = this.audio.calculateTimeout(intro_duration);
```

Comment:
We need to rewrite `validateAudioDuration` to return the duration on success, and then store that. 

The way this is currently implemented will calculate the duration twice per call, which is expensive.